helpers/handlebars.js

```diff
@@ -0,0 +1,76 @@
+const handelbarsHelpers = {
+  not: (v) => {
+    return !v;
+  },
+  
+  eq: (v1, v2) => {
+    return v1 === v2;
+  },
+  
+  ne: (v1, v2) => {
+    return v1 !== v2;
+  },
+  
+  lt: (v1, v2) => {
+    return v1 < v2;
+  },
+  
+  gt: (v1, v2) => {
+    return v1 > v2;
+  },
+  
+  lte: (v1, v2) => {
+    return v1 <= v2;
+  },
+  
+  gte: (v1, v2) => {
+    return v1 >= v2;
+  },
+  
+  and: (v1, v2) => {
+    return v1 && v2;
+  },
+  
+  or: (v1, v2) => {
+    return v1 || v2;
+  },
+  
+  startwith: (str, substr) => {
+    if (!str) {
+      return false;
+    }
+    return str.startsWith(substr);
+  },
+  
+  mstartwith: () => {
+    var args = Array.prototype.slice.call(arguments);
```

Comment:
* Obsolete and overly complicated use of `arguments`, a spread operator should be used instead
* Array.some should be preferred, as for this use it's simpler than reduce, and it stops scanning the array as soon as a searched value is found

```javascript
mstartwith: (str, ...args) => {
    if (!str) {
      return false;
    }

    return args.some(arg => str.startsWith(arg));
}
```